calculate_statistics.py

```diff
@@ -1,36 +1,38 @@
-class CalculateStatistics:
+#!/usr/bin/env python
+# Copyright (c) 2022 Madeleine Dunn. All rights reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Script for calculating job statistics."""
 
+
+class CalculateStatistics:
     def __init__(self):
-        self.number_of_instances = 0
         self.number_of_passes = 0
         self.number_of_fails = 0
         self.number_of_skips = 0
         self.number_of_cancellations = 0
         self.average_queued_duration_list = []
         self.average_duration_list = []
 
-    def calculate_instances(self):
-        self.number_of_instances += 1
-
-    def collect_average_durations(self, av_duration, duration_type):
-        if duration_type == "queued":
-            if av_duration == "None":
-                pass
-            else:
-                self.average_queued_duration_list.append(float(av_duration))
-        elif duration_type == "duration":
-            if av_duration == "None":
-                pass
-            else:
-                self.average_duration_list.append(float(av_duration))
+    def collect_average_durations(self, average_duration, duration_type):
+        """Collect and store average durations from the data store."""
+        if duration_type == "queued" and average_duration != "None":
```

Comment:
```suggestion
        if duration_type == "queued" and average_duration:
```

Does the suggestion do the job?

Reply:
Yes it does.